staging/src/k8s.io/apiserver/pkg/storage/cacher/cacher.go

```diff
@@ -359,6 +359,8 @@ func NewCacherFromConfig(config Config) (*Cacher, error) {
 
 	watchCache := newWatchCache(
 		config.KeyFunc, cacher.processEvent, config.GetAttrsFunc, config.Versioner, config.Indexers, config.Clock, objType)
+	objectType := objType.String()
+	watchCacheCapacity.WithLabelValues(objectType).Set(float64(cacher.watchCache.capacity))
```

Comment:
I guess I want clear.
Please move this to "newWatchCache" function:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/storage/cacher/watch_cache.go#L218

Reply:
ok, i have  moved this to "newWatchCache" function